### DISCUSSION TITLE
请问为什么动画之类的资源文件采用的是json格式文件，而不是gltf等通用的格式标准加json描述？

### DISCUSSION THREAD

**Discussion by rocketman123456** [BoomingTech/Piccolo#261] [Q&A] [answered]
如题

**Comment by Lurenlzm** [comment]
看起来是因为 引擎直接支持 Json 的序列化，使用Json 可能更加方便一些

**Comment by rocketman123456** [comment]
主要是之后是不是需要我们自己处理才能添加新的asset，感觉会比较麻烦。也许会做一些工具，来导出这些数据到引擎的json文件中？

**Comment by hyv1001** [comment, marked as answer]
一般引擎都有自己的文件格式，这样可以和引擎更好的结合，特别是在序列化，反射等方面更加灵活自由。
对于gltf，fbx等标准格式的支持，一般通过插件进行导入导出，小引擎后续也有这方面的规划。
